Rename ImageQueryOptions as AvatarQueryOptions

pinarol opened the pull request:
Closes https://github.com/Automattic/Gravatar-SDK-iOS/issues/114

### Description

- Rename ImageQueryOptions as AvatarQueryOptions
- Rename forceDefaultImage as forceDefaultAvatar
- Rename forceDefaultImageSwitchWithLabel as forceDefaultAvatarSwitchWithLabel

### Testing Steps

CI green.

I also checked with WPiOS repo and we don't need any changes on that side.

pinarol committed:
Rename DefaultImageOption as DefaultAvatarOption

pinarol committed:
Replace "Default Image" with "Default Avatar"

pinarol committed:
Replace "default image" with "default avatar"

pinarol committed:
Rename ImageQueryOptions as AvatarQueryOptions

pinarol committed:
Merge branch 'trunk' into wppinar/rename_ImageQueryOptions

# Conflicts:
#	Tests/GravatarTests/GravatarURLTests.swift

pinarol committed:
Rename forceDefaultImage as forceDefaultAvatar

pinarol committed:
Update demo view controller

pinarol committed:
Rename the file

etoledom reviewed (approved):
Looking good! Thank you for this 🙏 

I did a project search for `defaultimage` and there are a few places in the Demo app and in tests where we still refer to the default avatar as `defaultimage`. Specifically:

- preferredDefaultImage (Demo app)
- testFetchImageWithDefaultImageOption (AvatarServiceTests)
- testUrlWithDefaultImage (GravatarURLTests)
- urlAddingDefaultImage (GravatarURLTests)
- testDefaultImageOptionIsSet (GravatarWrapper_UIImageViewTests)

In the SDK is all good, so I'm approving anyway. Feel free to merge when ready 👍

pinarol committed:
Do more replacements

preferredDefaultImage (Demo app)
testFetchImageWithDefaultImageOption (AvatarServiceTests)
testUrlWithDefaultImage (GravatarURLTests)
urlAddingDefaultImage (GravatarURLTests)
testDefaultImageOptionIsSet (GravatarWrapper_UIImageViewTests)

pinarol commented:
Thanks a lot! I addressed those items.
